Formula/djl-serving.rb

```diff
@@ -0,0 +1,59 @@
+class DjlServing < Formula
+  desc "This module contains an universal model serving implementation"
+  homepage "https://github.com/awslabs/djl/tree/master/serving"
+  url "https://djl-ai.s3.amazonaws.com/publish/djl-serving/serving-0.11.0.tar"
+  sha256 "93dfd5e4efca5208918b016338209de37b96796af6dc584e269093c116f04030"
+  license "Apache-2.0"
+
+  bottle :unneeded
+
+  depends_on "openjdk"
+
+  def install
+    # Install files
+    rm_rf Dir["bin/*.bat"]
+    mv "bin/serving", "bin/djl-serving"
+    libexec.install Dir["*"]
+    (bin/"djl-serving").write_env_script "#{libexec}/bin/djl-serving", Language::Java.overridable_java_home_env
+  end
+
+  plist_options manual: "djl-serving"
+
+  def plist
+    <<~EOS
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+          <key>Disabled</key>
+          <false/>
+          <key>Label</key>
+          <string>#{plist_name}</string>
+          <key>ProgramArguments</key>
+          <array>
+            <string>#{bin}/djl-serving</string>
+            <string>run</string>
+          </array>
+          <key>KeepAlive</key>
+          <true/>
+        </dict>
+      </plist>
+    EOS
+  end
+
+  test do
+    port = free_port
+    (testpath/"config.properties").write <<~EOS
+      inference_address=http://127.0.0.1:#{port}
+      management_address=http://127.0.0.1:#{port}
+    EOS
+    ENV["MODEL_SERVER_HOME"] = testpath
+    cp_r Dir["#{libexec}/*"], testpath
```

Comment:
Why do we need this line? Is it because the test will attempt to write into `libexec` for logging otherwise?

Also, are you able to make sure that this writes logs into `#{var}/log` rather than into `libexec`?

Reply:
Yes, we have made sure the logs aren't written to `libexec`. I can check the effect of removing line 51.

Reply:
What about during regular use of `djl-serving`, outside of the `test` block? Where does it write its logs?

Reply:
By default, it writes the logs to `libexec/logs` folder. It depends on the `MODEL_SERVER_HOME` environment variable.

Reply:
That's what I was asking earlier. It shouldn't be writing there. Instead, it should be writing into `#{var}/log`. We can wrap `djl-serving` in a wrapper that sets the `MODEL_SERVER_HOME` variable in a way that can be overridden by the user.

Just add it to the `env` argument in your `write_env_script` call.

Reply:
So it'll be `(bin/"djl-serving", MODEL_SERVER_HOME).write_env_script "#{libexec}/bin/djl-serving", Language::Java.overridable_java_home_env`

Reply:
It probably has to be something like
```ruby
env = { MODEL_SERVER_HOME: "${MODEL_SERVER_HOME:-#{var}/log}" }
env.merge!(Language::Java.overridable_java_home_env)
(bin/"djl-serving").write_env_script "#{libexec}/bin/djl-serving", env
```
Where does `djl-serving` store its logs relative to `MODEL_SERVER_HOME`? Does it store it in `MODEL_SERVER_HOME`, or does it use something like `${MODEL_SERVER_HOME}/logs`?

Reply:
It's inside `${MODEL_SERVER_HOME}/logs`

Reply:
Then the first line should probably just be
```ruby
env = { MODEL_SERVER_HOME: "${MODEL_SERVER_HOME:-#{var}}" }
```
Does that seem right to you, @SMillerDev? 

Or, I guess this might also work:
```ruby
env = { MODEL_SERVER_HOME: "${MODEL_SERVER_HOME:-#{var}/djl-serving}" }
```

Reply:
The `env = { MODEL_SERVER_HOME: "${MODEL_SERVER_HOME:-#{var}}" }` seems to be the way in macos in general.

Reply:
I guess we can go with that then.